admin/app/controllers/authenticated/organizations/get/invitations.js

```diff
@@ -39,7 +40,12 @@ export default class InvitationsController extends Controller {
       this.notifications.success(`Un email a bien a été envoyé à l'adresse ${organizationInvitation.email}.`);
       this.userEmailToInvite = null;
     } catch (err) {
-      this.errorResponseHandler.notify(err, this.CUSTOM_ERROR_MESSAGES);
+      const errorDetail = get(err, 'errors[0].detail');
+      if (errorDetail) {
```

Comment:
Je comprends qu'on veuille afficher un message custom avec l'adresse email non valide mais faire `this.notifications.error(errorDetail);` et afficher le détail d'une erreur du back c'est un peu trop "permissif". 
Si on a pas bien catché les erreurs en back, on risque d'afficher une erreur qu'on ne voulait (classique : erreur de db avec éventuellement plein de détails techniques genre le path qu'on aimerait pas trop divulger).

on peut faire ça : 
```js
    } catch (err) {
      this.CUSTOM_ERROR_MESSAGES.STATUS_400 = `Échec lors de l'envoi d'un e-mail à l'adresse ${email} car le domaine semble invalide.`;
      this.errorResponseHandler.notify(err, this.CUSTOM_ERROR_MESSAGES);
    }
  ```

Reply:
Je suis d'accord mais je n'aime pas trop cette solution non plus car on peut avoir un statut 400 pour d'autres raisons. Je vais voir comment faire mieux, avec un code erreur peut-être.

Reply:
Si on met un code spécifique pour cette erreur côté back, je crois qu'on pourra faire ensuite aussi 
     
      this.CUSTOM_ERROR_MESSAGES.MON_CODE_QUE_JE_CONNAIS_RENVOYE_PAR_LE_BACK = `Échec lors de l'envoi d'un e-mail à l'adresse ${email} car le domaine semble invalide.`;
      
Dans un cas comme dans l'autre, j'ai l'impression que ça sous tend qu'on connaisse les erreurs renvoyées par l'appel api

Reply:
Effectivement, ça implique qu'on connaisse les erreurs qui peuvent être renvoyées par l'API, mais ça me paraît obligatoire si on veut avoir un message d'erreur précis et ne pas utiliser directement le message renvoyé par l'API. L'intérêt d'utiliser un code plutôt que le statut est qu'on peut avoir un code spécifique "INVALID_DOMAIN" alors que le statut HTTP 400 peut correspondre à beaucoup d'autres choses.

Du coup, j'ai proposé quelque chose en ce sens, mais sans utiliser le `errorResponseHandler` qui ne gère que les statuts HTTP.

Reply:
Tout à fait d'accord sur l'utilisation de codes précis.
Pour le errorResponseHandler, ça permet d'uniformiser nos erreurs et pas faire du code custom partout. Je serai plutôt pour le mettre à jour du coup

Reply:
J'aime bien l'idée de gérer les codes erreurs dans le `errorResponseHandler`, j'ai fait ça.